src/cli/install.sh

```diff
@@ -160,6 +160,32 @@ then
         echo -e "  $BWhite export BUN_INSTALL=\"$bun_install\"$Color_Off"
         echo -e "  $BWhite export PATH=\"\$BUN_INSTALL/bin:\$PATH\"$Color_Off"
     fi
+elif 
+    test $(basename $SHELL) == "bash"
+then 
+    # Install completions, but we don't care if it fails
+    IS_BUN_AUTO_UPDATE="true" SHELL="bash" $exe completions >/dev/null 2>&1
+
+    if test -w $HOME/.bashrc; then
+        echo -e "\n# bun\nexport BUN_INSTALL=\"$bun_install\"" >>"$HOME/.bashrc"
+        echo -e "export PATH=\"\$BUN_INSTALL/bin:\$PATH\"" >>"$HOME/.bashrc"
+        echo ""
+        echo -e "$Dim Added \"$bin_dir\" to \$PATH in \"~/.bashrc\"$Color_Off"
+
+        echo ""
+        echo -e "To get started, run"
+        echo -e "$BWhite"
```

Comment:
If possible, it could be nice to use the new loggers, `info`, `info_bold`... cf :https://github.com/oven-sh/bun/blob/main/src/cli/install.sh#L45